tests/dragonfly/seeder/__init__.py

```diff
@@ -139,9 +139,9 @@ def __init__(
         collection_size=None,
         types: typing.Optional[typing.List[str]] = None,
         huge_value_percentage=1,
-        huge_value_size=1024,
-        # 1 huge entries per container/key as default
-        huge_value_csize=1,
+        huge_value_size=512,
+        # 2 huge entries per container/key as default
+        huge_value_csize=2,
```

Comment:
does this configuration means that 1% of keys has 2 huge entries in the container?

Reply:
yes!